proto/buf/alpha/registry/v1alpha1/organization.proto

```diff
@@ -189,3 +193,33 @@ message RemoveOrganizationMemberRequest {
 }
 
 message RemoveOrganizationMemberResponse {}
+
+message GetOrganizationSettingsRequest{
+  // The ID of the organization for which to get the settings.
+  string organization_id = 1;
+}
+
+message GetOrganizationSettingsResponse {
+  RepositoryRole repository_role = 1;
+  PluginRole plugin_role = 2;
+  TemplateRole template_role = 3;
+}
+
+message UpdateOrganizationSettingsRequest {
+  message RepositoryRoleMessage {
+    RepositoryRole role = 1;
+  }
+  message PluginRoleMessage {
+    PluginRole role = 1;
+  }
+  message TemplateRoleMessage {
+    TemplateRole role = 1;
+  }
+  // The ID of the organization for which to update the base roles.
+  string organization_id = 1;
+  RepositoryRoleMessage repository_role = 2;
+  PluginRoleMessage plugin_role = 3;
+  TemplateRoleMessage template_role = 4;
```

Comment:
Could you add a comment to each of these fields explaining that they are optional? If none is set, no update is made.